components/menu/index.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+export {Menu} from './menu';
+export {SubMenu} from './sub_menu';
+export {MenuItem, styled} from './menu_item';
```

Comment:
Why do we need to re-export `styled` from here? If it's because of `PopoverMenuItem`, then perhaps it should live in this folder as well

Reply:
Yes following your suggestion i moved `PopoverMenuItem` back to menu directory.

Reply:
Sounds good.

This also made me realize that, with the common components using MUI, if we want to give people using that library the ability to customize stuff, it'll be really hard to do without leaking details about MUI. Even if we didn't have to expose `styled`, we'd probably need to have them overwrite MUI's CSS classes. I'll bring it up with Los Tigres as well